src/clarity-angular/forms/_select.clarity.scss

```diff
@@ -73,5 +72,20 @@
                 border-color: $clr-form-field-border-color;
             }
         }
+
+        //Multiple Select
+
+        &.multiple select[multiple] {
```

Comment:
Why do we need an extra class on the `.select` wrapper?

It doesn't seem necessary. Can't we just say `select[multiple]` here?